python/ray/air/config.py

```diff
@@ -119,6 +121,9 @@ class ScalingConfig:
         placement_strategy: The placement strategy to use for the
             placement group of the Ray actors. See :ref:`Placement Group
             Strategies <pgroup-strategy>` for the possible options.
```

Comment:
Should we have some warning in the docstring about how the colocation above will not happen if you specify a custom placement strategy like `SPREAD`/`STRICT_SPREAD`?

Reply:
no longer apply here after we force colocate trainer and rank 0 worker